.github/workflows/tests.yml

```diff
@@ -95,3 +95,12 @@ jobs:
         with:
           name: bottles
           path: "*.bottle.*"
+  label-pr:
+    name: Label pr pr-pull
+    needs: test-bot
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions-ecosystem/action-add-labels@v1
+        if: always() && github.event_name == 'pull_request'
```

Comment:
What's with the 'always()' is that needed?

Reply:
I think maybe not needed, I just went off how the homebrew workflow worked https://github.com/momentohq/homebrew-tap/blob/main/.github/workflows/tests.yml#L93. I can remove it